util/BUILD

```diff
@@ -7,6 +7,7 @@ cc_library(
         "string.h",
     ],
     visibility = ["//visibility:public"],
+    deps = ["@range-v3//:range-v3"],
```

Comment:
This could be simplified to `@ranges-v3`. If the package name matches the dependency's name, it's not needed!